### ISSUE TITLE
Module update-status from API

### ISSUE THREAD

**Issue by sindrebroch** [Jopyth/MMM-Remote-Control#276]
### Description

Hi.

I would love to be able to check if there are any updates available per module through the API. As far as I can tell it's not possible as of now? I see that it is possible check mmUpdateAvailable, and to run an update-command, but I would love to be able to see state as well, without running the update-command.

I'm writing an integration for HomeAssistant, and update-control would be a great addition.

Great work on the API, it's fantastic!

**Comment by ezeholz** [comment]
You're right, there isn't any check-update. I'm gonna work on it. Thanks!

**Comment by sindrebroch** [comment]
Not super comfortable with express-apis, but I threw together something in a fork. There is also a bunch of formatting changes, so the diff is a bit messy. 

I guess you can use it for inspiration if you want to add it at some point:
https://github.com/sindrebroch/MMM-Remote-Control
